avaframe/runCom1DFA.py

```diff
@@ -33,7 +33,7 @@
 
 startTime = time.time()
 # Run Standalone DFA
-reportDictList = com1DFA.runSamos(cfg, avalancheDir)
+reportDictList = com1DFA.runCom1DFA(cfg, avalancheDir)
```

Comment:
This is very confusing. There is a runCom1DFA.py and a runCom1DFA function in com1DFA/com1DFA.py. I got a bit lost trying to figure out what was happening.